drivers/generate_stage2.R

```diff
@@ -1,20 +1,27 @@
-source("https://raw.githubusercontent.com/eco4cast/neon4cast/ci_upgrade/R/to_hourly.R") # is this branch stable?
+source("https://raw.githubusercontent.com/eco4cast/neon4cast/ci_upgrade/R/to_hourly.R") # is this branch stable? why use ci_upgrade ?
```

Comment:
Curious why this isn't merged into `main` and if we're going to rely on the `ci_upgrade` branch

Reply:
because I did the merge on Jan 1, 2024